### DISCUSSION TITLE
New tag for bulk @method / @property

### DISCUSSION THREAD

**Discussion by thattomperson** [phpDocumentor/phpDocumentor#3659] [Ideas]
can we add a way to add methods from a class that is not extended

example

```
/**
 * @inhert-methods Parent
 */
class Child {
    protected $parent;

    public function __class($name, $arguments) {
        return call_user_func_array([$this->parent, $name], $arguments);
    }
}
```

**Comment by mvriel** [comment]
This sounds a lot like the `@mixin` proposition: https://github.com/phpDocumentor/fig-standards/issues/35, does that seem applicable to you?

**Comment by thattomperson** [comment]
Thanks I wasn't quite sure where to find this and I didn't know that phpstorm already had this as I use phpDocumentor docs to find out what I can use in phpstorm

just had a quick read
from my understanding `@mixin` brings everything in, including public properties on the `Parent` class which in my example would not be relevant / misdocumented

although having everything is probably better than having nothing
